metering/metering-installing-metering.adoc

```diff
@@ -5,5 +5,99 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Installing metering stub
+On {product-title} 4 clusters the Metering Operator can be installed through the OperatorHub. If you are using an earlier version of OpenShift or a Kubernetes cluster without OpenShift, use the link:https://github.com/operator-framework/operator-metering/tree/master/Documentation[upstream documentation] to install, configure, and use the Metering Operator.
 
+include::modules/metering-install-prerequisites.adoc[leveloffset=+1]
+
+include::modules/metering-install-operator.adoc[leveloffset=+1]
+
+[id="metering-install-metering-stack_{context}"]
+== Installing the Metering stack 
+
+To install the Metering stack create a metering resource and add your configuration details. Review xref:../metering/metering-configuring-metering.adoc#configuring-storage[configuring storage for metering] for more details about all available configuration options. At a minimum you need to configure persistent storage to hold reporting data, and the Hive metastore to hold metadata about database tables managed by Presto and Hive.
+
+After entering you configuration details, the the metering resource installs all the necessary metering components inside the metering namespace.
+
+[IMPORTANT]
+====
+There can only be one Metering resource in the namespace containing the metering-operator. Any other configuration is not supported.
+====
+
+.Procedure
+
+. From the web console, ensure you are on the Operator Details page for the Metering operator in the Metering project. You can navigate to this page by clicking *Operators* -> *Installed Operators*, then selecting the Metering operator.
+
+. Under *Provided APIs*, click *Create Instance* on the Metering card. This opens a YAML editor with the default configuration file where you can define your Metering configuration.
```

Comment:
It should be the `MeteringConfig` card for 4.2 (this may need updating elsewhere)

Reply:
That's a better name. It felt weird installing the metering operator then creating a "Metering" resource. I'll also do a sanity check once we have 4.2 OLM available.

Reply:
I'll have something easier to test 4.2 OLM based installs sometime next week.